### ISSUE TITLE
SignalR client written in Kotlin

### ISSUE THREAD

**Issue by xiazunyang** [SignalR/SignalR#4681]
this issue not a bug.
I wrote a SignalR client in kotlin, take a look if you need it.

https://github.com/xiazunyang/signalr-lite

If you think I'm doing a good job, can you link to the signalr-lite project somewhere?

**Comment by sachinthanapm** [comment]
Could you please explain how to use this on a project?

**Comment by xiazunyang** [comment]
> @xiazunyang Could you please explain how to use this on a project?

Yes, here it is: https://github.com/xiazunyang/signalr-lite/tree/master/example/signalr-lite-demo
